lib/solidus_retail/order/generate_pos_order.rb

```diff
@@ -45,17 +45,37 @@ def create_order
 
```

Comment:
I didn't realize in your last PR that everything was put inside the `lib` folder instead of the `app` folder. 

I would like if the code would follow the directory structure that is on master. This may be outside to scope of this PR but I wanted to flag it at least.